addon/-private/system/relationships/state/relationship.js

```diff
@@ -266,6 +269,41 @@ export default class Relationship {
     this.flushCanonicalLater();
   }
 
+  /*
+    Call this method once a record deletion has been persisted
+    to purge it from both current and canonical state of all
+    relationships.
+
+    @method removeDeletedInternalModelFromInverse
+    @private
+   */
+  removeDeletedInternalModelFromInverse(internalModel) {
```

Comment:
there isn't one afaik, just seemed better to pass this in with the original implementation. using `this.internalModel` is probably a bit safer